<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WoT_CH</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="layoutMode">
     <enum>QListView::SinglePass</enum>
    </property>
   </widget>
   <widget class="QLabel" name="showTile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Tile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_ChangesModify">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Перемещение, изменение значений клавишами&lt;/p&gt;&lt;p&gt;W, A, S, D - шаг изменения можно указать в меню.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>Изменять</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="rb_setPosition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Изменение позиции тайла на атласе.&lt;/p&gt;&lt;p&gt;Ориентируйтесь по миникарте&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Позиция</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_setSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Изменение размера тайла на атласе.&lt;/p&gt;&lt;p&gt;Ориентируйтесь по рамке на миникарте&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Размер</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_setCenter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Настройка центра тайла.&lt;/p&gt;&lt;p&gt;Смотрите на сам тайл&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Смещение</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="numHorizont">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Установка активного параметра по горизонтали&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="minimum">
      <number>-100000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="numVertical">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Установка активного параметра по вертикали&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="minimum">
      <number>-100000</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>listWidget</zorder>
   <zorder>gb_ChangesModify</zorder>
   <zorder>showTile</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="menu_open"/>
    <addaction name="menu_save"/>
    <addaction name="menu_save_as"/>
   </widget>
   <widget class="QMenu" name="menuTEX">
    <property name="title">
     <string>TEX</string>
    </property>
    <addaction name="menu_texPNG"/>
    <addaction name="menu_texWEBP"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Подстройка</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTEX"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_open">
   <property name="text">
    <string>Открыть txt</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menu_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="menu_save_as">
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="menu_texPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="menu_texWEBP">
   <property name="text">
    <string>WEBP</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Шаг инкремента</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
